Resources/translations/SonataNewsBundle.hu.xliff

```diff
@@ -0,0 +1,396 @@
+                <source>form.comment.name</source>
+                <target>Név</target>
+            </trans-unit>
+            <trans-unit id="form.comment.email">
+                <source>form.comment.email</source>
+                <target>Email</target>
+            </trans-unit>
+            <trans-unit id="form.comment.url">
+                <source>form.comment.url</source>
+                <target>Url</target>
+            </trans-unit>
+            <trans-unit id="form.comment.message">
+                <source>form.comment.message</source>
+                <target>Üzenet</target>
+            </trans-unit>
+            <trans-unit id="view_all_posts">
+                <source>view_all_posts</source>
+                <target>Összes bejegyzés megtekintése</target>
+            </trans-unit>
+            <trans-unit id="post_is_public">
+                <source>post_is_public</source>
+                <target>Publikus</target>
+            </trans-unit>
+            <trans-unit id="post_is_not_public">
+                <source>post_is_not_public</source>
+                <target>Privát</target>
+            </trans-unit>
+            <trans-unit id="label_comment_valid">
+                <source>label_comment_valid</source>
+                <target>Érvényes</target>
+            </trans-unit>
+            <trans-unit id="label_comment_invalid">
+                <source>label_comment_invalid</source>
+                <target>Érvénytelen</target>
+            </trans-unit>
+            <trans-unit id="label_unknown_status">
+                <source>label_unknown_status</source>
+                <target>Ismeretlen</target>
+            </trans-unit>
+            <trans-unit id="label_unknown_moderate">
+                <source>label_unknown_moderate</source>
+                <target>Mérsékelt</target>
+            </trans-unit>
+            <trans-unit id="view_all_comments">
+                <source>view_all_comments</source>
+                <target>Összes hozzászólás megtekintése</target>
+            </trans-unit>
+            <trans-unit id="sonata_news_archive_breadcrumb">
+                <source>sonata_news_archive_breadcrumb</source>
+                <target>Hírek</target>
+            </trans-unit>
+            <trans-unit id="group_general">
+                <source>group_general</source>
+                <target>Általános</target>
+            </trans-unit>
+            <trans-unit id="group_comment">
+                <source>group_comment</source>
+                <target>Hozzászólás</target>
+            </trans-unit>
+            <trans-unit id="form.label_image">
+                <source>form.label_image</source>
+                <target>Kép</target>
+            </trans-unit>
+            <trans-unit id="sidemenu.link_edit_post">
+                <source>sidemenu.link_edit_post</source>
+                <target>Szerkesztés</target>
+            </trans-unit>
+        </body>
+    </file>
+</xliff>
```

Comment:
Sorry, I edited with Counterparts, missed checking the new line. It should be fine now.